python/oneflow/test/modules/test_sort.py

```diff
@@ -84,12 +84,29 @@ def test_sort_with_random_data(test_case):
         y = torch.sort(x, dim=random(low=-4, high=4).to(int), descending=random_bool())
         return y[0], y[1]
 
+    @autotest(n=5, auto_backward=False, check_graph=True)
+    def test_sort_return_values_with_random_data_(test_case):
```

Comment:
```suggestion
    def test_sort_return_types_with_random_data_(test_case):
```
这里建议改成 return_types，因为这系列返回值的类型都是 `oneflow.return_types.xxx`。如果叫成 values，一方面可能让人想不到是测 return_types，另一方面可能会让人误以为是测试结果的数值正确性